packages/components/src/tools-panel/stories/index.tsx

```diff
@@ -112,23 +135,27 @@ export const _default = () => {
 		</PanelWrapperView>
 	);
 };
+Default.args = {
+	label: 'Tools Panel (default example)',
```

Comment:
Ideally we should pass `children` to `ToolsPanel` via the `children` prop in the `args` object. But given how intertwined `ToolsPanel` and `ToolsPanelItem` components are in these stories, for now I've decided to keep them as they were